sys/cheri/cheric.h

```diff
@@ -82,6 +83,12 @@
 /* Increment @p dst to have the address of @p src */
 #define cheri_copyaddress(dst, src)	(cheri_setaddress(dst, cheri_getaddress(src)))
 
+/*
+ * XXX-AM: Ugly thing to make this importable from linux compat,
+ * which redefines __always_inline
```

Comment:
Since this code is only compiled by clang, we could always expand __always_inline. However, I don't think always_inline is really needed here. We compile at -O2 so this should always be inlined anyway.

Reply:
I'm happy to ditch __always_inline if it ends up being useless, will remove it